src/com/owncloud/android/lib/resources/files/DownloadRemoteFileOperation.java

```diff
@@ -160,9 +161,12 @@ protected int downloadFile(OwnCloudClient client, File targetFile) throws HttpEx
                     // TODO some kind of error control!
                 }
 
-            } else {
+            } else if (status != 403){
```

Comment:
Maybe you can define it as a constant:
private static final int ERROR_FORBIDDEN = 403;